rootfs/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.5
+RUN apt-get upgrade && apt-get install -y \ 
+  && curl -sSL https://raw.githubusercontent.com/pypa/pip/7.1.2/contrib/get-pip.py | python - \
+  && pip install flake8 
+WORKDIR /tmp
```

Comment:
@krancour changes made to the run and workdir..I guess user will mount a volume with his content..and there is no need for /tmp